frontend/src/domain/types/reporting.ts

```diff
@@ -1,27 +1,47 @@
+import type { SeaFront } from '../entities/alerts/constants'
 import type { PendingAlertValue } from './alert'
 
-// export enum ReportingType {
-//   ALERT = 'ALERT',
-//   INFRACTION_SUSPICION = 'INFRACTION_SUSPICION',
-//   OBSERVATION = 'OBSERVATION'
-// }
+export enum ReportingType {
+  // TODO Should be renamed 'PENDING_ALERT'.
+  ALERT = 'ALERT',
+  INFRACTION_SUSPICION = 'INFRACTION_SUSPICION',
+  OBSERVATION = 'OBSERVATION'
+}
 
-export type Reporting<Value = PendingAlertValue | InfractionSuspicion | Observation> = {
+export type BaseReporting = {
   creationDate: string
   externalReferenceNumber: string
   id: string
   internalReferenceNumber: string
   ircs: string
-  // TODO Doesn't exists.
-  // type: ReportingType
   underCharter: boolean
   validationDate: string
-  // TODO Create a specific type with a discriminator prop to avoid type-guessing issues
-  value: Value
   vesselIdentifier: string
   vesselName: string
+
+  // TODO These 2 props shouldn't be there at all and should be treated in a separated redux state.
```

Comment:
Give the `id` when editing reporting in `ReportingList.tsx` : `edit(editingIsDisabled, reporting)`